.travis.yml

```diff
@@ -6,162 +6,166 @@ addons:
     update: false
 
 stages:
-  - "EDA Tools - Misc"
-  - "EDA Tools - Bitstream"
-  - "EDA Tools - Simulation"
-  - "EDA Tools - Synthesis"
-  - "EDA Tools - PnR"
-  - "EDA Tools - Formal"
-  - "EDA Tools - Xilinx Vivado metapackages"
+  - name: "No dependencies" # Doesn't depend on anything apart from upstream packages
+    if: type != cron
+  - name: "Has first level dependencies" # Depends on a package in the "no dependencies" group
+    if: type != cron
+  - name: "Main label upload" # Puts Anaconda packages to the main label
+    if: type = push AND \
+        branch = master
```

Comment:
Maybe we should push to a label named after the branch?

Reply:
travis-{branch}-{run}? Sure, makes sense